src/pages/release-announce/1.5.0.md

```diff
@@ -0,0 +1,187 @@
+1.24 through 1.29 are supported at runtime.
+
+### Reproducible build artifacts
+
+Building YuniKorn from source code now generates reproducible build artifacts
+which depend only on the version of YuniKorn built and the go compiler version
+used. The `README.md` file in the official YuniKorn source release contains a
+list of binaries generated and their expected SHA-512 hashes. This can be used
+to verify that a binary artifact claiming to be from the YuniKorn project is
+authentic. See
+[YUNIKORN-2419](https://issues.apache.org/jira/browse/YUNIKORN-2419) for
+details.
+
+### Web UI enhancements
+
+YuniKorn 1.5.0 includes a number of updates to the Web UI:
+
+- [YUNIKORN-1362](https://issues.apache.org/jira/browse/YUNIKORN-1362)
+  Allow node list filtering based on selector
+- [YUNIKORN-2088](https://issues.apache.org/jira/browse/YUNIKORN-2088)
+  Node utilization chart updates
+- [YUNIKORN-2214](https://issues.apache.org/jira/browse/YUNIKORN-2214)
+  Add Sorting functionality to the Allocation view
+- [YUNIKORN-2414](https://issues.apache.org/jira/browse/YUNIKORN-2414)
+  Support air-gapped clusters by embedding fonts and icons
+
+### Improved Prometheus metric grouping
+
+YuniKorn queue metrics are now exposed via labels on a common
+`yunikorn_queue_app` metric with separate `queue` labels for each queue.
+The existing `yunikorn_{queue_name}_queue_app` metrics are still generated,
+but will be **deprecated** in YuniKorn 1.6.0 and **removed** in YuniKorn 1.7.0.
+See [YUNIKORN-970](https://issues.apache.org/jira/browse/YUNIKORN-970) for
+further details.
+
+### Revamped scheduler initialization support
+
+YuniKorn 1.5.0 has dramatically simplified the logic used during early
+startup to reconcile existing allocations with YuniKorn's internal
+state. The result is faster, more stable initialization and better
+consistency with the Kubernetes API server. See
+[YUNIKORN-2099](https://issues.apache.org/jira/browse/YUNIKORN-2099).
+
+### Better allocation traceability
+
+YuniKorn internally tracks asks and allocations by a unique ID. In YuniKorn
+1.5.0, the ask ID (typically derived from the Pod UID in Kubernetes) will
+be used as the basis for the allocation ID as well. A future enhancement is
+planned to unify asks and allocations into a single object for even more
+simplified tracking. See
+[YUNIKORN-2204](https://issues.apache.org/jira/browse/YUNIKORN-2204) for
+information regarding the YuniKorn 1.5.0 changes and
+[YUNIKORN-2457](https://issues.apache.org/jira/browse/YUNIKORN-2457) for
+future enhancements.
+
+### REST API enhancement to node utilization
+
+The YuniKorn REST API has been enhanced to support the retrieval of
+node utilization statistics for multiple resource types in addition to
+the existing support for dominant resource type. This allows for more
+expressive visualizations in the YuniKorn Web UI. See
+[YUNIKORN-2257](https://issues.apache.org/jira/browse/YUNIKORN-2257) for
+details.
+
```

Comment:
how about adding https://issues.apache.org/jira/browse/YUNIKORN-2235 ? That brings new and useful restful APIs.

Reply:
Good catch. Added in latest commit.